packages/bezier-react/src/components/ListItem/ListItem.styled.ts

```diff
@@ -137,6 +145,9 @@ export const Wrapper = styled.div<StyledWrapperProps>`
     ${focused && css`
       ${FocusedItemStyle}
 
+      ${StyledLegacyIcon} {
```

Comment:
```suggestion
      ${StyledIcon},
      ${StyledLegacyIcon} {
```

이 편이 중복을 줄일 수 있어서 좋을 거 같아요.